README.md

```diff
@@ -1,2 +1,19 @@
-# nature_news_disparities
-Analysis pipeline for Nature news articles
+# Analysis Pipeline for Nature News Articles
+
+Note: This is currently a work in progress -- many things will change and may also be undocumented at this moment.
+
+The code is made of 2 parts currently: 
+1. Scrape Nature News Articles
+2. Analyze the text of the articles to identify possible speaker biases
+
+## Scraping
+  
+The code relies upon [scrapy](https://docs.scrapy.org/en/latest/index.html) to crawl links and process the articles.
+  This code is found in [here](https://github.com/nrosed/nature_news_disparities/tree/main/nature_news_scraper), and most of which is automatically generated.
+  To run the scraper tool, you run the shell script found here: [run_scrape_gold_standard.sh](https://github.com/nrosed/nature_news_disparities/blob/main/nature_news_scraper/run_scrape_gold_standard.sh).
+  This runs an initial scrape to identify all articles in 2020, 2015, and 2010, then randomly chooses 10 from each year to write to a file. 
+  This will later be extended.
```

Comment:
Are you going to scale up to all articles between 2010-2020?

Reply:
Eventually I will scale up all articles from 2001 - 2020. This is just to make a benchmark dataset that I will hand annotate.